clang/test/CheckedCRewriter/checkedptrret.c

```diff
@@ -0,0 +1,22 @@
+// RUN: cconv-standalone -addcr  %s -- | FileCheck -match-full-lines --check-prefixes="CHECK" %s
```

Comment:
Can you run these tests with `-alltypes` too? Have a look at some of the tests @sroy4899 just checked in, which use `CHECK`, `CHECK_ALL`, and `CHECK_NOALL` together to indicate the results with and without `-alltypes` on.

Reply:
Sure